org.lflang/src/org/lflang/FileConfig.java

```diff
@@ -439,32 +507,115 @@ public static void copyFile(Path source, Path destination)  throws IOException {
      * 
      *  @param source The source file as a path relative to the classpath.
      *  @param destination The file system path that the source file is copied to.
+     *  @param skipIfUnchanged If true, don't overwrite the destination file if its content would not be changed
      * @throws IOException If the given source cannot be copied.
      */
-    public void copyFileFromClassPath(String source, Path destination) throws IOException {
+    public void copyFileFromClassPath(final String source, final Path destination, final boolean skipIfUnchanged) throws IOException {
         InputStream sourceStream = this.getClass().getResourceAsStream(source);
 
         // Copy the file.
-        try (sourceStream) {
-            if (sourceStream == null) {
-                throw new IOException(
-                    "A required target resource could not be found: " + source + "\n" +
-                        "Perhaps a git submodule is missing or not up to date.\n" +
-                        "See https://github.com/icyphy/lingua-franca/wiki/downloading-and-building#clone-the-lingua-franca-repository.\n"
-                        +
-                        "Also try to refresh and clean the project explorer if working from eclipse.");
+        if (sourceStream == null) {
+            throw new IOException(
+                "A required target resource could not be found: " + source + "\n" +
+                    "Perhaps a git submodule is missing or not up to date.\n" +
+                    "See https://github.com/icyphy/lingua-franca/wiki/downloading-and-building#clone-the-lingua-franca-repository.\n"
```

Comment:
```suggestion
                    "To initialize, run `git submodule init`; to update, run `git submodule update`.\n"
```

Reply:
(We might want `git submodule update --init` with `--` before the "init.")

Reply:
I agree that in a situation like this `git submodule update --init` might be a preferable shorthand. I'll adjust it.